src/IntelliTect.Coalesce.Tests/Util/StringExtensionsTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using IntelliTect.Coalesce.Utilities;
+using Xunit;
+
+namespace IntelliTect.Coalesce.Tests.Util
```

Comment:
We already have a StringExtension(s)Tests class in the test project. The Util directory in the test project is meant for utilities for the tests themselves, not tests of Coalesce's utility classes and methods.

If you're feeling generous, you could take this opportunity to get rid of the `AddUnique` methods from the string extensions and replace the 3 usages of them with `HashSet<string>`, since that's probably what that functionality should have been in the first place. This would let you get rid of the single tests that is in StringExtensionTests now.

Reply:
Done